src/hooks/use-all-media.js

```diff
@@ -0,0 +1,43 @@
+import { useEffect, useState } from 'react';
+import fetchTwitchVideos from '../utils/fetch-twitch-videos';
+import fetchYoutubeData from '../utils/fetch-youtube-data';
+import fetchLybsinPodcasts from '../utils/fetch-lybsin-podcasts';
+
+const YT_PLAYLIST = 'PLN3n1USn4xlmyw3ebYuZmGp60mcENitdM';
+
+const useAllMedia = () => {
+    const [media, setMedia] = useState(null);
+    const [error, setError] = useState(null);
+    const [isLoading, setIsLoading] = useState(false);
+
+    useEffect(() => {
+        const getMedia = async () => {
+            setIsLoading(true);
+            try {
+                const youtubeVideos = fetchYoutubeData(YT_PLAYLIST);
+                const twitchVideos = fetchTwitchVideos();
+                const lybsinPodcasts = fetchLybsinPodcasts();
+
+                const allMedia = await Promise.all([
+                    youtubeVideos,
+                    twitchVideos,
+                    lybsinPodcasts,
+                ]);
+                const mediaList = allMedia.flat(1);
```

Comment:
Sorry missed this the last time but the default value for `depth` is 1 so we don't actually need this arg. I also didn't do this in `use-event-data`, oops!